contrib/fedora-default-kernel/default-kernel.service

```diff
@@ -3,4 +3,4 @@ Description=linux-surface default kernel watchdog
 
 [Service]
 Type=oneshot
-ExecStart=/bin/sh -c "grubby --set-default /boot/vmlinuz*surface*"
+ExecStart=/bin/sh /etc/systemd/system/default-kernel.sh
```

Comment:
I would kinda prefer if this was in `/usr/local/bin`.

Also this new script should be added to the instructions in the readme.

Reply:
Also probably a better idea to add a `chmod +x` to the README and not bypass the shebang (e.g. `#!/bin/sh`) here.

Reply:
I did try to execute it directly, but got
```
default-kernel.service: Failed at step EXEC spawning /etc/systemd/system/default-kernel.sh: Permission denied
```

Reply:
As discussed on Matrix: This was a SELinux error.

Reply:
Moved the script to `/usr/local/bin`, added it to the readme, and called it directly instead of using the shebang.